cmp/ag-grid/AgGridModel.js

```diff
@@ -293,10 +297,16 @@ export class AgGridModel {
     getColumnState() {
         this.throwIfNotReady();
 
-        const {agColumnApi} = this;
+        const {agColumnApi} = this,
+            columns = agColumnApi.getColumnState();
+
+        // sort config is collected in the getSortState function
+        const sortKeys = ['sort', 'sortIndex'];
+        columns.forEach(col => sortKeys.forEach(it => delete col[it]));
```

Comment:
ok to modify the object coming back from getColumnState?  assume so.

Reply:
Yes, the object is generated fresh on function call, not a reference.

https://github.com/ag-grid/ag-grid/blob/3ed885a9cde6742d5c2469093f02b424d9c2bb05/community-modules/core/src/ts/columnController/columnController.ts#L1708
https://github.com/ag-grid/ag-grid/blob/3ed885a9cde6742d5c2469093f02b424d9c2bb05/community-modules/core/src/ts/columnController/columnController.ts#L1682